plugins/SitesManager/templates/_vueTabInstructions.twig

```diff
@@ -0,0 +1,22 @@
+<p>{{ 'SitesManager_SiteWithoutDataVueIntro'|translate }}</p>
+<br>
+<p>{{ 'SitesManager_SiteWithoutDataCloudflareFollowStepsIntro'|translate }}</p>
```

Comment:
```suggestion
<p>{{ 'SitesManager_SiteWithoutDataVueFollowStepsIntro'|translate }}</p>
```
same as we discussed the other day.. we either should have Vue specific string with the same content or a reusable string for all different no-data options.

For now might be easier to go with Vue specific with the same content.

Reply:
@michalkleiner I would rather make this generic in some other PR not in this one as we still have to port this PR back to 4.x-dev